### PR TITLE
Bug/fix bypass scope

### PR DESCRIPTION
**Pull request by charlie-pisuraj**
Covers regeneration of the pb.go files,
Fix AuthScope function map names and 1 Incorrect Scope value

### CODE REVIEW

File: pkg/dss/server.go [hunk 1]
```diff
@@ -29,17 +29,17 @@ type Server struct {
 func (s *Server) AuthScopes() map[string][]string {
 	return map[string][]string{
 		"GetIdentificationServiceArea":     []string{ReadISAScope},
-		"PutIdentificationServiceArea":     []string{WriteISAScope},
+		"CreateIdentificationServiceArea":  []string{WriteISAScope},
 		"DeleteIdentificationServiceArea":  []string{WriteISAScope},
-		"PutSubscription":                  []string{ReadISAScope},
+		"CreateSubscription":               []string{WriteISAScope},
```
Comment (steeling) [inline pkg/dss/server.go]: wheres update?

Comment (charlie-pisuraj) [reply pkg/dss/server.go]: oh you are right, let me fix that

**Comment by charlie-pisuraj** [comment]
ping @vosst when you get a chance can you review this

**Comment by vosst** [comment]
> ping @vosst when you get a chance can you review this

Yup, on it.

**Pull request merged**